pkg/middleware/openidconnect.go

```diff
@@ -92,6 +91,8 @@ func OpenIDConnect(opts ...ocisoidc.Option) func(next http.Handler) http.Handler
 			// The claims we want to have
 			var claims ocisoidc.StandardClaims
 
+			token := header[7:]
```

Comment:
I guess you are cutting of `Bearer ` here? Maybe something like `strings.TrimPrefix(header, "Bearer ")` or `header[len("Bearer "):]` would be clearer.

Reply:
Thanks :)